packages/apollo-engine-reporting-protobuf/src/reports.proto

```diff
@@ -4,6 +4,12 @@ package mdg.engine.proto;
 
 import "google/protobuf/timestamp.proto";
 
+import "google/protobuf/descriptor.proto";
```

Comment:
I think my PR description that implemented this explained that you need to leave this stuff out of the version of the file you use in protobufjs somehow, because protobufjs doesn't need the option to be declared but adding the import will massively inflate the size of generated code. Bah.

Reply:
Yep. I need to fix this.

Reply:
This is still happening